### PR TITLE
Create initial contributing docs

### PR DESCRIPTION
**Pull request by HBBisenieks**
Per suggestions in the chat and possibly previous PRs, here's at least the start of a contributing guide for both code and docs. Code section is basically just cherry-picked from the main README, while docs section is whole-cloth based on my review of the docs and comments on previous work, especially #350.

I would love to see this doc expanded in the future, especially the code section, which I can't speak to from first-hand experience yet.

### CODE REVIEW

**Comment by HBBisenieks** [comment]
also i know the commit history looks like trash right now; i'm working on fixing it, but dayjob has demanded most of my attention

**Comment by kdb424** [comment]
Honestly, before a final merge, just squash it all and it's fine about history. I'll get to reviewing.

**Comment by kdb424** [comment]
Would also like to see some mention of running `make fix-formatting fix-isort test` before committing/pushing to limit the amount of times that I have to state that. Looks great so far.

**Comment by HBBisenieks** [comment]
@kdb424 changed up the wording in the code section to specifically call out `make fix-isort...`

**Comment by Tonasz** [comment]
Maybe it is worth to add some licensing notes? The home page readme got such paragraph:
> All software in this repository is licensed under the GNU Public License, verison 3. All documentation and hardware designs are licensed under the Creative Commons Attribution-ShareAlike 4.0 license. Contributions to this repository must use these licenses unless otherwise agreed to by the Core team.

Especially the last sentence seems to be important for contributions.
btw: In the future, try using rebase to limit number of KMKfw:master merges :)

**Comment by HBBisenieks** [comment]
@Tonasz just committed the license section pulled from the main readme

**Review by kdb424 — APPROVED** [review]
@Tonasz can give a final review since they requested changes. It's good in my book though. Will merge when they are happy. Thanks for this!

**Comment by HBBisenieks** [comment]
real happy to be able to contribute docs! <3

**Review by Tonasz — APPROVED** [review]
Ofc I approve :)

File: docs/contributing.md
Comment (yswallow) [inline docs/contributing.md:18]: typo of `fix-isort` ?

Comment (HBBisenieks) [reply docs/contributing.md:18]: ope, you're right

**Comment by HBBisenieks** [comment]
looks like we just need a merge from one of the maintainers?

**Comment by kdb424** [comment]
I've been pretty busy with other things. I keep an eye out any time I have time, but I'm basically the only one actively merging until one of the other two reviewers accept those abilities. I'm trying.

**Comment by HBBisenieks** [comment]
sorry, @kdb424 didn't want to put more pressure on you; just wasn't sure if you'd seen. thanks for the work you've been doing to keep this project going!

**Pull request merged**
